tests/foreman/remote_DB/test_remote_DB.py

```diff
@@ -0,0 +1,228 @@
+"""Test class for Remote DB feature
+
+:Requirement: Offload DBs from Satellite Server
+
+:CaseAutomation: ManualOnly
+
+:CaseLevel: Integration
+
+:CaseComponent: ForemanMaintain
```

Comment:
@lvrtelov It should be External Db component, if tests are related to Foreman-mainatain it should got to https://github.com/SatelliteQE/testfm

Reply:
@ntkathole it can't, because there is no such component in Bugzilla. We want CaseComponent to be aligned with components in bugzilla.

I agree it's better to put these tests into testfm, if it's possible to automate them there.